projects/plugins/jetpack/modules/google-fonts.php

```diff
@@ -110,24 +110,35 @@ function jetpack_add_google_fonts_provider() {
 	$fonts_to_register = apply_filters( 'jetpack_google_fonts_list', JETPACK_GOOGLE_FONTS_LIST );
 
 	foreach ( $fonts_to_register as $font_family ) {
-		wp_register_webfonts(
-			array(
-				array(
-					'font-family'  => $font_family,
-					'font-weight'  => '100 900',
-					'font-style'   => 'normal',
-					'font-display' => 'fallback',
-					'provider'     => 'jetpack-google-fonts',
-				),
-				array(
-					'font-family'  => $font_family,
-					'font-weight'  => '100 900',
-					'font-style'   => 'italic',
-					'font-display' => 'fallback',
-					'provider'     => 'jetpack-google-fonts',
-				),
-			)
+		$fonts = array();
+
+		$font_italic = array(
+			'font-family'  => $font_family,
+			'font-weight'  => '100 900',
+			'font-style'   => 'normal',
+			'font-display' => 'fallback',
+			'provider'     => 'jetpack-google-fonts',
+		);
+
+		$font_normal = array(
+			'font-family'  => $font_family,
+			'font-weight'  => '100 900',
+			'font-style'   => 'italic',
+			'font-display' => 'fallback',
+			'provider'     => 'jetpack-google-fonts',
 		);
+
+		// New WP Fonts API format since Gutenberg 14.9 requires keyed array
+		// See https://github.com/Automattic/jetpack/issues/28063
+		if ( class_exists( 'WP_Fonts' ) ) {
```

Comment:
Will work only after https://github.com/WordPress/gutenberg/pull/46749 is merged.